src/kibana/components/vis/vis.js

```diff
@@ -70,7 +70,7 @@ define(function (require) {
       if (_.isString(this.type)) this.type = visTypes.byName[this.type];
 
       this.listeners = _.assign({}, state.listeners, this.type.listeners);
-      this.params = _.defaults({}, state.params || {}, this.type.params.defaults || {});
+      this.params = _.defaults({}, _.cloneDeep(state.params || {}), this.type.params.defaults || {});
```

Comment:
I needed to do a `_.cloneDeep` to get access to object values in the params. Are you asking because _.clone is better?